build-tools/packages/build-cli/docs/typetestDetails.md

```diff
@@ -103,11 +103,16 @@ the baseMajor version is `${v.major}.0.0` and the baseMinor version is `${v.majo
 
 The "previous" versions work similarly, but the major/minor/patch segment is reduced by 1. That is, for a version _v_,
 the previousMajor version is `${min(v.major - 1, 1)}.0.0`, the previousMinor version is
-`${v.major}.${min(v.minor - 1, 0)}.0`, and the previousPatch is `${v.major}.${v.minor}.${min(v.patch - 1, 0)}.0`.
+`${v.major}.${min(v.minor - 1, 0)}.0`, and the previousPatch is `${v.major}.${v.minor}.${min(v.patch - 1, 0)}`.
 
 The "previous" versions never roll back below 1 for the major version and 0 for minor and patch. That is, the
 previousMajor, previousMinor, and previousPatch versions for `1.0.0` are all `1.0.0`.
 
+> **Warning**
+>
+> `^previousMinor` allows the "previous" version to jump into the same range as the baseline, so use with caution.
+> In most cases, you probably want to use `~previousMinor`.
```

Comment:
@tylerbutler given the complexities when thinking about these version specifiers, I thought at least a warning for now is probably a good idea.

Reply:
Good idea, but I prefer not to use blockquote since it's not semantically correct here. Maybe:

```suggestion
**Warning**: `^previousMinor` allows the baseline (the "previous" version) to jump into the same range as the current version, so use with caution.
In most cases, you probably want to use `~previousMinor`.
```

Also corrected the baseline terminology.

Reply:
What's the semantical issue? I used the blockquote for the [warning block formatting](https://github.com/community/community/discussions/16925):

<img width="156" alt="image" src="https://user-images.githubusercontent.com/716334/203445016-c0b9e1cb-3ddb-4b2b-a9db-c9f10bf43501.png">

Not super attached to it, but want to understand what's the issue you see with it.

Reply:
See this comment in that discussion thread: https://github.com/community/community/discussions/16925#discussioncomment-2830438

The HTML spec defines the blockquote element thus: "The [blockquote](https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) a section that is quoted from another source."

You'll see a number of people on that thread saying, please don't change the semantic meaning of blockquotes. Now, as the discussion points out, if GitHub decides to render that markdown as semantically correct HTML, that's fine, but as of now it looks like `blockquote` elements are still output. And if they do go this route, IMO they would do better with a syntax more like this: https://github.com/community/community/discussions/16925#discussioncomment-2791869

Reply:
Ah, I wasn't aware it actually *renders* as a `<blockquote>`, yeah, that's not great. Changing that.